products/rockylinux.md

```diff
@@ -24,15 +23,15 @@ identifiers:
 releases:
 -   releaseCycle: "9"
     releaseDate: 2022-07-14
-    support: 2025-05-31
+    support: 2027-05-31
     eol: 2032-05-31
     latest: "9.2"
     latestReleaseDate: 2023-05-16
```

Comment:
```suggestion
    latestReleaseDate: 2023-05-20
```

According to https://wiki.rockylinux.org/rocky/version/#current-supported-releases